java/src/jmri/jmrix/mrc/MrcSystemConnectionMemo.java

```diff
@@ -195,7 +195,7 @@ public MrcClockControl getClockControl() {
     /*public MrcConsistManager  getConsistManager() { return consistManager; }*/
 
     protected ResourceBundle getActionModelResourceBundle() {
-        return ResourceBundle.getBundle("jmri.jmrix.mrc.MrcActionListBundle"); //IN18N
+        return ResourceBundle.getBundle("jmri.jmrix.mrc.MrcBundle"); //IN18N
```

Comment:
I'm seeing a few things that should be in the bundle in that class so I'll add those and change the flags.

So a line doing the Bundle.getMessage() should have //NOI18N saying it (the key) doesn't get translated but if there is text being put into something on screen should be //IN18N, saying it likely should be translated??
-ken c